setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name="Emojirades PlusPlus",
-    version="0.7.1",
+    version="0.7.2",
```

Comment:
If you can just leave this at the current version, I increment this myself on the commit that performs the release.

Reply:
done